### PR TITLE
Updated OAuth Reset Information

### PR DESCRIPTION
**Pull request by tatze96**

### CODE REVIEW

File: protocols/dropbox.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ You can connect to multiple accounts at the same time. Create a new bookmark for
 
 ### OAuth Reset
 
-If you have accidentally logged in with the wrong Dropbox Account or want to change the login of the Dropbox bookmark you can reset the OAuth token by deleting the entries related to `https://Dropbox (user)@api.dropboxapi.com` out of the *Windows Credential Manager* or on macOS out of *Keychain Access.app*.
+If you have accidentally logged in with the wrong Dropbox Account or want to change the login of the Dropbox bookmark you can reset the OAuth token by deleting the entries related to `duck:dropbox?user=(user)` out of the *Windows Credential Manager* or on macOS out of *Keychain Access.app*.
```
Comment (dkocher) [inline protocols/dropbox.md:26]: On macOS the entries in the keychain are named `api.dropboxapi.com`.

File: protocols/google_cloud_storage.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ You access the page with the authorization code from the link displayed in the l
 
 #### OAuth Reset
 
-If you have accidentally logged in with the wrong Google Cloud Storage username or want to change the login of the Google Cloud Storage bookmark you can reset the OAuth token by deleting the entries related to `https://Google Cloud Storage (user)` out of the *Windows Credential Manager* or on macOS out of *Keychain Access.app*.
+If you have accidentally logged in with the wrong Google Cloud Storage username or want to change the login of the Google Cloud Storage bookmark you can reset the OAuth token by deleting the entries related to `duck:googlecloudstorage?user=(user)` out of the *Windows Credential Manager* or on macOS out of *Keychain Access.app*.
```
Comment (dkocher) [inline protocols/google_cloud_storage.md:40]: On macOS the entries in the keychain are named `accounts.google.com`.

File: protocols/google_drive.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ You can connect to multiple accounts at the same time. Create a new bookmark for
 
 ### OAuth Reset
 
-If you have accidentally logged in with the wrong Google Drive Account or want to change the login of the Google Drive bookmark you can reset the OAuth token by deleting the entries related to `https://Google Drive (user)@accounts.google.com` out of the *Windows Credential Manager* or on macOS out of *Keychain*.
+If you have accidentally logged in with the wrong Google Drive Account or want to change the login of the Google Drive bookmark you can reset the OAuth token by deleting the entries related to `duck:googledrive?user=(user)` out of the *Windows Credential Manager* or on macOS out of *Keychain*.
```
Comment (dkocher) [inline protocols/google_drive.md:39]: On macOS the entries in the keychain are named `accounts.google.com`.

File: protocols/onedrive.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ You can connect to multiple accounts at the same time. Create a new bookmark for
 
 ### OAuth Reset
 
-If you have accidentally logged in with the wrong OneDrive Account or want to change the login of the OneDrive bookmark you can reset the OAuth token by deleting the entries related to `https://Microsoft OneDrive (user)@login.microsoft.com` out of the *Windows Credential Manager* or on macOS out of *Keychain Access.app*.
+If you have accidentally logged in with the wrong OneDrive Account or want to change the login of the OneDrive bookmark you can reset the OAuth token by deleting the entries related to `duck:onedrive?user=(user)` out of the *Windows Credential Manager* or on macOS out of *Keychain Access.app*.
```
Comment (dkocher) [inline protocols/onedrive.md:29]: On macOS the entries in the keychain are named `login.microsoftonline.com `.

**Review by dkocher — CHANGES_REQUESTED** [review]
It should not be required to manually delete saved tokens from the keychain or credentials manager. Instead, creating a new bookmark will start a new authentication flow allowing the user to login with a different user.

**Review by Cassie205 — APPROVED** [review]

File: protocols/dropbox.md [hunk 5]
```diff
@@ -23,7 +23,9 @@ You can connect to multiple accounts at the same time. Create a new bookmark for
 
 ### OAuth Reset
 
-If you have accidentally logged in with the wrong Dropbox Account or want to change the login of the Dropbox bookmark you can reset the OAuth token by deleting the entries related to `https://Dropbox (user)@api.dropboxapi.com` out of the *Windows Credential Manager* or on macOS out of *Keychain Access.app*.
+If you have accidentally logged in with the wrong Dropbox Account or want to change the login of the Dropbox bookmark delete the current bookmark and create a new one to start a new authentication flow.
+
+Alternatively, you can reset the OAuth token by deleting the entries related to `duck:dropbox?user=(user)` out of the *Windows Credential Manager* or on macOS the entries related to `https://Dropbox (user)@api.dropboxapi.com` out of *Keychain Access.app*.
```
Comment (dkocher) [inline protocols/dropbox.md:28]: On macOS the entries can be found by _Account_ named `Dropbox (Email) OAuth2 Access Token` or _Name_ `api.dropboxapi.com`.

**Review by dkocher — CHANGES_REQUESTED** [review]

**Review by dkocher — APPROVED** [review]

**Pull request merged**